README.md

```diff
@@ -1,35 +1,80 @@
-# HomestakerOS
+# 🚀 HomestakerOS
 
-HomestakerOS is a web UI which creates custom Linux OS for Ethereum homestaking. It aims to democratize homestaking by simplifying the process of creating and maintaining servers in home environments.
+HomestakerOS is a web UI that creates a custom Linux OS for Ethereum homestaking. It aims to democratize homestaking by simplifying the process of creating and maintaining servers in home environments.
 
-The wizard produces Linux disk images based on NixOS. NixOS allows configurations to be public, deterministic, and self-upgrading. Further, by loading the whole operating system into the RAM, we can eliminate the works on my machine tantrum, while also making it possible to be booted by double-clicking a kernel execution script -- and if you want to return to your previous distribution, just restart your computer.
+This is still a work in progress. We are actively seeking funding and support for ongoing development!
 
```

Comment:
a screenshot and url to demo here

Reply:
Done. I think the screenshot would be too much, since the link is right there